dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java

```diff
@@ -186,6 +190,26 @@ private static void populateJarsFromWlp(File pomXml) throws Exception {
         }
     }
 
+    public static int runTCKMvnCmd(LibertyServer server) throws Exception {
+        if (!init) {
+            init(server);
+        }
+        // Everything under autoFVT/results is collected from the child build machine
+        File mvnOutput = new File(MvnUtils.logDir, "mvnOutput_TCK");
+        int rc = runCmd(MvnUtils.mvnCliTckRoot, MvnUtils.tckRunnerDir, mvnOutput);
+        Assert.assertEquals("mvn returned non-zero: " + rc, 0, rc);
+        File src = new File(MvnUtils.logDir, "tck/surefire-reports/junitreports");
+        File tgt = new File(MvnUtils.logDir, "junit");
```

Comment:
So logDir must be 'results' for this to work - so better to call it 'resultsDir none of these are logs after all.